client/app/translations.js

```diff
@@ -11,7 +11,8 @@ angular.module('gettext').run(['gettextCatalog', function (gettextCatalog) {
         "KEYWORDS": "TAGS",
         "Keywords": "Tags",
         "URGENCY": "RANKING",
-        "Urgency": "Ranking"
+        "Urgency": "Ranking",
+        "'SLUGLINE is a required field'": "Keyword is a required field"
```

Comment:
Yes it seems like. Or the maybe the key is actually `'SLUGLINE is a required field'`.
But it works fine, I merge it :)